src/core/tsi/ssl_transport_security.cc

```diff
@@ -144,6 +144,7 @@ struct tsi_ssl_frame_protector {
 static gpr_once g_init_openssl_once = GPR_ONCE_INIT;
 static int g_ssl_ctx_ex_factory_index = -1;
 static const unsigned char kSslSessionIdContext[] = {'g', 'r', 'p', 'c'};
+static int g_ssl_ex_ca_cert_index = -1;
```

Comment:
Here and throughout: we seem to always refer to "ca cert", but really we're interested in the root CA cert that was used to verify the chain. Can we rename instead say (e.g.) "verified root cert"? For example, this variable would become `g_ssl_ex_verified_root_cert_index`?